### PR TITLE
Update docs (text fixes, sample regions json)

### PR DESCRIPTION
**Pull request by hlomzik**
Generated automatically, all the fixes should go to https://github.com/heartexlabs/label-studio-frontend/pull/236

### CODE REVIEW

File: docs/source/tags/ellipselabels.md [hunk 1]
```diff
@@ -13,16 +13,45 @@ Used only for Image
...
+| original_height | <code>number</code> | height of the original image (px) |
+| image_rotation | <code>number</code> | rotation degree of the image (deg) |
+| value | <code>Object</code> |  |
+| value.x | <code>number</code> | x coord of the top left corner before rotation (0-100) |
+| value.y | <code>number</code> | y coord of the top left corner before rotation (0-100) |
+| value.radiusX | <code>number</code> | radius by x axis (0-100) |
+| value.radiusY | <code>number</code> | radius by y axis (0-100) |
+| value.rotation | <code>number</code> | rotation degree (deg) |
+
+### Example JSON```js
```
Comment (smoreface) [inline docs/source/tags/ellipselabels.md:41]: typo here --> 
needs to be

### Example JSON

```json

not 
### Example JSON```js

File: docs/source/tags/hypertext.md [hunk 2]
```diff
@@ -6,23 +6,51 @@ meta_title: Hypertext Tags for Hypertext Markup (HTML)
...
+| Name | Type | Description |
+| --- | --- | --- |
+| value | <code>Object</code> |  |
+| value.start | <code>string</code> | xpath of the container where the region starts (xpath) |
+| value.end | <code>string</code> | xpath of the container where the region ends (xpath) |
+| value.startOffset | <code>number</code> | offset inside start container |
+| value.endOffset | <code>number</code> | offset inside end container |
+| [value.text] | <code>string</code> | text content of the region, may be missed |
+
+### Example JSON```js
```
Comment (smoreface) [inline docs/source/tags/hypertext.md:34]: must be a typo in the conversion script

**Review by smoreface — CHANGES_REQUESTED** [review]
Need to fix the script to properly encode example JSON and also update based on latest changes in [#236](https://github.com/heartexlabs/label-studio-frontend/pull/236/files)

**Comment by hlomzik** [comment]
> Need to fix the script to properly encode example JSON

I used `js` because it looks a bit better than `json` formatting. But looks good anyway and semantic is better, so changed it, thanks. And fixed script

**Review by smoreface — APPROVED** [review]
looks good, thanks for the fixes!

**Pull request merged**